docs/ref/csrf.txt

```diff
@@ -222,24 +222,36 @@ The CSRF protection is based on the following things:
 1. A CSRF cookie that is set to a random value (a session independent nonce, as
    it is called), which other sites will not have access to.
 
-   This cookie is set by ``CsrfViewMiddleware``.  It is meant to be permanent,
-   but since there is no way to set a cookie that never expires, it is sent with
-   every response that has called ``django.middleware.csrf.get_token()``
-   (the function used internally to retrieve the CSRF token).
+   This cookie is set by ``CsrfViewMiddleware``.  It is sent with every
+   response that has called ``django.middleware.csrf.get_token()`` (the
+   function used internally to retrieve the CSRF token), if it wasn't already
+   set on the request.
+
+   In order to protect against `BREACH`_ attacks, the token is not simply the
+   nonce; a random pad is added to the nonce and used to scramble it.
+
+.. _BREACH: http://breachattack.com/
```

Comment:
please move this line to the bottom of the section -- as it is now it causes incorrect rendering.